=========================================================================================================
Customized exception handling by using try catch
=========================================================================================================
It is highly recommended to handle exceptions.
The code which may rise an exception is called risky code and we have to define that code inside
a try block and corresponding handling code we have to define inside catch block.
try
{
    risky code 
}
catch(Exception e)
{
    handling code
}

//Program without try-catch

    public class p6
    {
        public static void main(String[] args) {
            System.out.println("Start");
            System.out.println(10/0);
            System.out.println("End");
        }
    }
    /*in above case we program will terminate abnormally

//Program with try-catch

    public class p6
    {
        public static void main(String[] args) {
            System.out.println("Start");
            try
            {
                System.out.println(10/0);
            }
            catch(ArithmeticException e)
            {
                System.out.println(10/2);
            }
            System.out.println("End");
        }
    }
    /* in above case program will not terminate normally as we are handling error by try-catch

